### PR TITLE
fixed number of symbols

### PR DESCRIPTION
**Pull request by Hunter2065**
# Checklist:

- [x] Added specs for all new ruby code
- [x] Green rspec run
- [x] Green rubocop
- [x] PR review from teammate

# What?
Changes for some ruby code

## Why?
'cause it's bad) 

## How?
changed number of symbols for correctly showing places' cards footer

## Testing?
yap)

## Screenshots (optional)
image

## Anything Else?
I change validation for title about 23 characters

### CODE REVIEW

**Review by RMuskovets — APPROVED** [review]
LGTM

**Comment by RMuskovets** [comment]
Actually.. some changes can be made =)

**Review by RMuskovets — APPROVED** [review]

File: app/views/places/_place.html.slim [hunk 1]
```diff
@@ -4,7 +4,7 @@
       .rating
         |⭐️4.9
       .card-footer
-        - if place.title.length < 16
+        - if place.title.length < 14
```
Comment (andrjuha01) [inline app/views/places/_place.html.slim:7]: Тут Точно щось не так

Comment (Hunter2065) [reply app/views/places/_place.html.slim:7]: ahahah, so, I think in future we can add validation for title, like title can't be infinity and for design I create some condition for styling elements)

Comment (VanyaNovak) [reply app/views/places/_place.html.slim:7]: we have validation for the title

Comment (Hunter2065) [reply app/views/places/_place.html.slim]: fixed)

**Review by andrjuha01 — APPROVED** [review]

**Pull request merged**